db/db_test2.cc

```diff
@@ -5625,6 +5696,7 @@ TEST_F(DBTest2, ChangePrefixExtractor) {
     ub = Slice(ub_str);
     {
       std::unique_ptr<Iterator> iterator(db_->NewIterator(ro));
+      ASSERT_OK(iterator->status());
```

Comment:
Please move to right before/after Valid check.

Reply:
Done.